modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/toolbarbuttons/plugin.js

```diff
@@ -22,70 +22,19 @@
 	CKEDITOR.plugins.add(pluginName, {
 		init(editor) {
 			editor.ui.addBalloonToolbarButton('ImageAlignLeft', {
-				click() {
-					const imageWidget = editor.widgets.selected[0];
-
-					if (imageWidget.name !== 'image') {
-						return;
-					}
-
-					imageWidget.focus();
-
-					const alignValue = imageWidget.data.align;
-
-					if (!alignValue || alignValue !== 'left') {
-						imageWidget.setData('align', 'left');
-					}
-					else {
-						imageWidget.setData('align', 'none');
-					}
-				},
+				command: 'justifyleft',
```

Comment:
Nice catch! Love to see so much unnecessary code removed :heart_eyes:

Reply:
![](https://media.giphy.com/media/ZJ7jAT03opd5XXCN52/giphy.gif)